src/lib/deployWorkers.ts

```diff
@@ -622,13 +633,89 @@ const validateWasmExist = async (
   }
 };
 
+const DEAL_TYPE: Deals[string] = {
+  dealId: "",
+  chainNetwork: "testnet",
```

Comment:
But that's not TYPE, it's a value 🤔 Why is it called `DEAL_TYPE`?

Reply:
`Deals[string]` is confusing, too

Reply:
I tried to explain this concept to you previously. I will try once again. I created a function that infers aqua types from js/json values - so this is the reason I have to encode type in a value - it's all typesafe and pretty convenient to use. We could have created typescript types to aqua types tool but that's a whole can of worms. Regarding `Deals[string]` it's not generic so it's not type-parametrized. It's a Record with string keys and Deals as values so in order to get the deal type in typescript I access Deals record with any string. It's just a way to avoid creating a separate `Deal` type, but I can do it so it's more readoble to the people who don't write typescript whole day long

Reply:
I just removed this name completely so it's not confusing and passed this directly to the function
(function has documentation and the argument to it is named properly so it's more clear what's that is)